### PR TITLE
 feat: compatibility with mobile device and hamburger menu for accessibility

### PR DESCRIPTION
**Pull request by moripen**
The hamburger menu is for the moment only an iconbutton that has no functionality.

### CODE REVIEW

File: src/components/container-with-background.tsx [hunk 1]
```diff
@@ -3,34 +3,46 @@ import { Box, Container } from 'theme-ui';
...
   color?: 'green' | 'beige';
   element?: 'element1' | 'element2' | 'element3' | 'element4' |'element5' | 'element6';
+  marginbottom?: '-170px';
   positionx?: 'right' | 'left';
-  positiony?: '0' | '10' | '20' | '25' | '30' | '32' | '40' |'50' | '60' |'70' | '80' |'90';
-  paddingtop?: '40px' | '190px' | '210px';
-  paddingbottom?: '210px';
+  positiony?: '0' | '10' | '20' | '25' | '30' | '32' | '40' | '44' | '46' | '50' | '60' |'70' | '80' |'90';
+  paddingtop?: '40px' | '50px' | '70px';
+  paddingbottom?: '90px';
```
Comment (cobraz) [inline src/components/container-with-background.tsx:10]: Maybe just pass the `sx` prop? Or use `React.forwardRef`?

File: src/components/layout/header.tsx [hunk 2]
```diff
@@ -30,17 +31,36 @@ export const Header: React.FC = () => {
...
+          <IconButton aria-label='Hamburger' sx={{
+            ml: 'auto',
+            display: ['block', 'none'],
+          }}
+          >
+             <svg width="32" height="22" viewBox="0 0 32 22" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M27 21.4336H0V18.2891H27V21.4336Z" fill="#0F2040"/>
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M32 12.2891H0V9.14453H32V12.2891Z" fill="#0F2040"/>
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M24 3.14459H0V0H24V3.14459Z" fill="#0F2040"/>
+            <path fill-rule="evenodd" clip-rule="evenodd" d="M32 3.14459H27V0H32V3.14459Z" fill="#0F2040"/></svg>
```
Comment (cobraz) [inline src/components/layout/header.tsx:57]: This SVG should probably be moved to a separte component, and converted to JSX components (e.g. `fill-rule` → `fillRule`)

File: src/pages/about.tsx [hunk 3]
```diff
@@ -8,7 +8,7 @@ const About: React.FC = () => {
   return (
     <Layout>
       <Grid sx={{ width: '100%' }}>
-        <ContainerWithBackground element='element2' paddingtop='190px' positionx='left' positiony='32' size='298'>
+        <ContainerWithBackground element='element2' paddingtop='50px' positionx='left' positiony='30' size='298' marginbottom='-170px'>
```
Comment (cobraz) [inline src/pages/about.tsx:11]: It's probably not safe to use CSS position if it's not absolutely necessary. If you'd like, we can go through the problem you're trying to fix together in a shared session?

File: src/pages/about.tsx [hunk 4]
```diff
@@ -78,12 +78,12 @@ const About: React.FC = () => {
...
               email={'markus@bjerk.io'}
               title={'Programvareutvikler'}
-              github={'https://github.com/CalculatedCode'}
+              github={'https://github.com/moripen'}
             />
           </Flex>
           </ContainerWithBackground>
 
-          <ContainerWithBackground element='element3' paddingbottom='210px' positionx='right' positiony='0'>
+          <ContainerWithBackground element='element3' positionx='right' positiony='0'>
```
Comment (cobraz) [inline src/pages/about.tsx:86]:
```suggestion
          <ContainerWithBackground element="element3" positionx="right" positiony="0">
```

File: src/components/container-with-background.tsx (hunk 1 above)
Comment (moripen) [reply src/components/container-with-background.tsx:10]: Yeah, that sounds a lot easier - but I still wanted to only let some values pass as viable.

**Review by cobraz — APPROVED** [review]
This looks really good. We probably want to refactor some parts of it, but I think this is good to land! ✈️

**Pull request merged**